com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -660,36 +657,45 @@ public int GetVideoMemoryCost()
 
         void RemoveCell(Cell cell)
         {
-            if (cell.loaded)
+            if (cells.TryGetValue(cell.index, out var cellInfo))
             {
-                if (cells.ContainsKey(cell.index))
-                    cells.Remove(cell.index);
+                cells.Remove(cell.index);
 
-                if (m_ChunkInfo.ContainsKey(cell.index))
-                    m_ChunkInfo.Remove(cell.index);
+                UnloadCell(cellInfo);
+            }
+        }
 
-                if (cell.flatIdxInCellIndices >= 0)
-                    m_CellIndices.MarkCellAsUnloaded(cell.flatIdxInCellIndices);
+        void UnloadCell(CellInfo cellInfo)
+        {
+            if (cellInfo.flatIdxInCellIndices >= 0)
+                m_CellIndices.MarkCellAsUnloaded(cellInfo.flatIdxInCellIndices);
 
-                RegId cellBricksID = new RegId();
-                if (m_CellToBricks.TryGetValue(cell, out cellBricksID))
-                {
-                    ReleaseBricks(cellBricksID);
-                    m_CellToBricks.Remove(cell);
-                }
-            }
+            ReleaseBricks(cellInfo);
+
+            cellInfo.loaded = false;
+            cellInfo.updateInfo = new ProbeBrickIndex.CellIndexUpdateInfo();
 
-            cell.loaded = false;
+            ClearDebugData();
```

Comment:
Why clearing the whole debug data?

Reply:
Currently this only clears probe debug (that's the only persistant part) but semantically yeah, for now we clear the whole thing.